release-notes/con-relnotes-techpreview-features.adoc

```diff
@@ -10,130 +10,14 @@ These features are for Technology Preview only. Technology Preview features are
-
-For more information about how to enable or disable or configure dynamic plugins, see the link:{LinkAdminGuide}#rhdh-installing-dynamic-plugins[Installing dynamic plugins] section in the _Administration guide for {product}_.
-
-.Red Hat supported dynamic plugins
-[cols="20%,40%,40%",options="header",subs="+attributes"]
-|===
-| Plugin
-| Plugin ID
-| Description
-
-|3scale|`@janus-idp/backstage-plugin-3scale-backend`|The 3scale Backstage provider plugin synchronizes the 3scale content into the catalog.
-
-|Ansible Automation Platform|`@janus-idp/backstage-plugin-aap-backend`| The Ansible Automation Platform (AAP) plugin synchronizes the accessible templates including job templates and workflow job templates from AAP into the catalog. 
-
-|Azure Container Registry|`@janus-idp/backstage-plugin-acr`|The Azure Container Registry (ACR) plugin displays information about your container images available in the Azure Container Registry.
-
-
-|Jfrog Artifactory|`@janus-idp/backstage-plugin-jfrog-artifactory`|The Jfrog Artifactory plugin displays information about your container images within the Jfrog Artifactory registry.
-
-|Nexus Repository Manager|`@janus-idp/backstage-plugin-nexus-repository-manager`|The Nexus Repository Manager plugin displays the information about your build artifacts that are available in the Nexus Repository Manager in your application.
-
-|ServiceNow|`@janus-idp/backstage-scaffolder-backend-module-servicenow`|This plugin provides template actions for ServiceNow.
-
-|SonarQube|`@janus-idp/backstage-scaffolder-backend-module-sonarqube`|This plugin provides template actions for SonarQube.
-
-|===
-
-[IMPORTANT]
-====
-Additional detail on how Red Hat provides support for bundled community dynamic plugins is available on the link:https://access.redhat.com/policy/developerhub-support-policy/[Red Hat Developer Support Policy] page.
-====
-
-.Community plugins that are validated with {product} {product-version}
-[cols="20%,40%,40%",options="header",subs="+attributes"]
-|===
-| Plugin
-| Plugin ID
-| Description
-
-|ArgoCD|`@roadiehq/scaffolder-backend-argocd`| Scaffolder backend plugin for ArgoCD.
-
-|Azure Devops|`@backstage/plugin-azure-devops`|Azure DevOps frontend plugin.
-
-|Azure Devops|`@backstage/plugin-azure-devops-backend`|Azure DevOps backend plugin that contains the API for retrieving builds and pull requests, that are used by the Azure DevOps frontend plugin.
-
-|Datadog|`@roadiehq/backstage-plugin-datadog`|This plugin embeds Datadog graphs and dashboards into {product-short}.
-
-|Dynatrace|`@backstage/plugin-dynatrace`|This plugin integrates towards Dynatrace.
-
-|GitHub|`@backstage/plugin-catalog-backend-module-github`|This plugin helps integrate towards GitHub.
+== {product} Operator
```

Comment:
The operator is not TP. It is going to be GA for 1.0

Reply:
given that
* https://issues.redhat.com/browse/RHIDP-1359  https://github.com/janus-idp/operator/issues/69 and
* this thread about SSL / external database support https://redhat-internal.slack.com/archives/C04CUSD4JSG/p1706793135543829?thread_ts=1706784752.312679&cid=C04CUSD4JSG and
* the fact that we have "alpha" API in https://github.com/janus-idp/operator/tree/main/api/v1alpha1

...  I think we're still shipping at Tech Preview in 1.1.

There was no operator in 1.0.

Reply:
@kadel, I see this comment from @nickboldt, which suggests that the Opertaor is going to be TP in RHDH 1.1 release. So, i am not making any mofifications.

Reply:
@rtaniwa Could you please confirm whether the Operaor is going TP or GA with this 1.1 release? Thanks

Reply:
it's also version 0.y in upstream https://github.com/janus-idp/operator/blob/main/Makefile#L6 which to me suggests "not yet GA".

Reply:
So, i am marking this conversation resolved. The Operator is not going GA this time.